Package.swift

```diff
@@ -10,12 +10,13 @@ let package = Package(
     .library(name: "PovioKit", targets: ["PovioKit"]),
     .library(name: "PovioKitNetworking", targets: ["PovioKitNetworking"]),
     .library(name: "PovioKitPromise", targets: ["PovioKitPromise"]),
-    .library(name: "PovioKitUI", targets: ["PovioKitUI"])
+    .library(name: "PovioKitUI", targets: ["PovioKitUI"]),
+    .library(name: "PovioKitAuthApple", targets: ["PovioKitAuthApple"]),
+    .library(name: "PovioKitAuthGoogle", targets: ["PovioKitAuthGoogle"]),
```

Comment:
Should we create a new submodule named `PovioKitAuth` and move these two inside?

Reply:
The reason for this decision was based on the fact, that we might have another provider in the future (Fb, Snapchat, Twitter, LinkedIn, etc). But good idea yes.

Reply:
Yep. I think it would be nicer if we have them all in a dedicated `Auth` submodule.

Reply:
Actually, we could go with that right now. And when a new provider came, we create a new product for that. So essentially, we'll have Apple in "core", and every other as separate product.

Reply:
> Yep. I think it would be nicer if we have them all in a dedicated `Auth` submodule.

The thing about having them all within the same product is dependencies. Imagine you only support Apple, but you'll have Facebook, Google, etc dependencies bundles within your app. Not good.

Reply:
Good point Toni. I think that scenario, where the app would support all providers is not impossible, but rather less frequent. I think your solution is doable, but not really sure about the usability. I would rather have smth like this:
```
Auth (core module with core features and Apple provider, since there is no dependency needed)
  - Google (google provider with dependency on core module)
  - Fb (FB provider with  dependency on core module)
  - ...
```

I would like to have "core" module with some extra logic later on for email, passwordless, etc auth as well.